### PR TITLE
feat: add options to customize [Embeddable]CatalogExtensionList

### PR DESCRIPTION
**Pull request by feloy**
Signed-off-by: Philippe Martin <phmartin@redhat.com>

### What does this PR do?

Add options to customize CatalogExtensionList and EmbeddableCatalogExtensionList

Will be helpful for #8989

### What issues does this PR fix or reference?

Part of #8571 

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/renderer/src/lib/extensions/CatalogExtensionList.svelte [hunk 1]
```diff
@@ -6,6 +6,8 @@ import type { CatalogExtensionInfoUI } from './catalog-extension-info-ui';
 import CatalogExtension from './CatalogExtension.svelte';
 
 export let catalogExtensions: CatalogExtensionInfoUI[];
+export let title: string = 'Available extensions';
+export let hideIfEmpty: boolean = false;
```
Comment (axel7083) [inline packages/renderer/src/lib/extensions/CatalogExtensionList.svelte]: `hideEmpty` is probably enough, I am not a big fan of having conditions in the naming,

other ideas would be `showEmptyScreen`, `displayEmptyScreen` or having a slot, and being empty by default

Comment (feloy) [reply packages/renderer/src/lib/extensions/CatalogExtensionList.svelte]: changed

**Review by axel7083 — APPROVED** [review]
LGTM

**Review by benoitf — COMMENTED** [review]
I think it might be easier with the `show` property rather than `hide`

`hideEmpty` --> `showEmpty`

so we have `if showEmpty` condition rather than a double not : `if !hideEmpty`

**Review by benoitf — COMMENTED** [review]
also at first I think it was something else so maybe add also the Screen to the property name

so we know it's about a screen that is displayed

**Review by benoitf — APPROVED** [review]

**Pull request merged**